ui/ducks/send/helpers.js

```diff
@@ -18,41 +19,28 @@ import {
   getAssetTransferData,
 } from '../../pages/send/send.utils';
 import { getGasPriceInHexWei } from '../../selectors';
-import { estimateGas } from '../../store/actions';
+import { estimateGasBuffered } from '../../store/actions';
 import { Numeric } from '../../../shared/modules/Numeric';
 
 export async function estimateGasLimitForSend({
   selectedAddress,
   value,
-  gasPrice,
   sendToken,
   to,
   data,
   isNonStandardEthChain,
   chainId,
   gasLimit,
-  ...options
 }) {
   let isSimpleSendOnNonStandardNetwork = false;
 
-  // blockGasLimit may be a falsy, but defined, value when we receive it from
-  // state, so we use logical or to fall back to MIN_GAS_LIMIT_HEX. Some
-  // network implementations check the gas parameter supplied to
-  // eth_estimateGas for validity. For this reason, we set token sends
-  // blockGasLimit default to a higher number. Note that the current gasLimit
-  // on a BLOCK is 15,000,000 and will be 30,000,000 on mainnet after London.
-  // Meanwhile, MIN_GAS_LIMIT_HEX is 0x5208.
-  let blockGasLimit = MIN_GAS_LIMIT_HEX;
-  if (options.blockGasLimit) {
-    blockGasLimit = options.blockGasLimit;
-  } else if (sendToken) {
-    blockGasLimit = GAS_LIMITS.BASE_TOKEN_ESTIMATE;
-  }
```

Comment:
`blockGasLimit` will fetched from provider in controller API `estimateGasBuffered`